netbox/extras/migrations/0043_jobresult.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 3.0.7 on 2020-06-23 02:28
+
+import uuid
+
+from django.conf import settings
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import extras.utils
+
+
+def convert_job_results(apps, schema_editor):
+    """
+    Convert ReportResult objects to JobResult objects
+    """
+    from django.contrib.contenttypes.management import create_contenttypes
+    from extras.choices import JobResultStatusChoices
+
+    ReportResult = apps.get_model('extras', 'ReportResult')
+    JobResult = apps.get_model('extras', 'JobResult')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    app_config = apps.get_app_config('extras')
+    app_config.models_module = app_config.models_module or True
+    create_contenttypes(app_config)
```

Comment:
I could not get it to work without doing this. It could not find the report content type, which is being created earlier in the migration. Perhaps we could split it out into two migrations?

Reply:
Let's split it into two migrations. I recall having strange problems in the past with migrations which both create a new model and then immediately create instances of that model. Best to keep it simple.